compiler/test/stdlib/array.test.gr

```diff
@@ -250,3 +250,20 @@ assert Array.unique([> 1, 1, 2, 2, 3, 3]) == [> 1, 2, 3]
 assert Array.unique([> 0, 1, 2]) == [> 0, 1, 2]
 assert Array.unique([> 0, 0, 0, 0, 0, 0]) == [> 0]
 assert Array.unique([>]) == [>]
+
+// Array.slice
+
+let  testChars = [> 'a', 'b', 'c']
+
+assert Array.slice(0, 1, testChars) == [> 'a']
+assert Array.slice(1, Array.length(testChars), testChars) == [> 'b', 'c']
```

Comment:
What's the expected behaviour for Array.slice(0,0,testChars) ?

Reply:
I believe it's an empty array. Let's add the test 😄 
```suggestion
assert Array.slice(1, Array.length(testChars), testChars) == [> 'b', 'c']
assert Array.slice(0, 0, testChars) == [>]
```